warehouse/query-core/src/test/java/datawave/query/tables/facets/FacetQueryTestTableHelper.java

```diff
@@ -0,0 +1,69 @@
+package datawave.query.tables.facets;
+
+import org.apache.accumulo.core.client.AccumuloException;
+import org.apache.accumulo.core.client.AccumuloSecurityException;
+import org.apache.accumulo.core.client.TableExistsException;
+import org.apache.accumulo.core.client.TableNotFoundException;
+import org.apache.accumulo.core.client.admin.TableOperations;
+import org.apache.accumulo.core.security.Authorizations;
+import org.apache.log4j.Logger;
+
+import datawave.helpers.PrintUtility;
+import datawave.ingest.mapreduce.handler.facet.FacetHandler;
+import datawave.ingest.table.config.FacetTableConfigHelper;
+import datawave.query.MockAccumuloRecordWriter;
+import datawave.query.QueryTestTableHelper;
+import datawave.query.RebuildingScannerTestHelper;
+
+public class FacetQueryTestTableHelper extends QueryTestTableHelper {
+    public static final String FACET_TABLE_NAME = "facet";
+    public static final String FACET_HASH_TABLE_NAME = "facetHash";
+    public static final String FACET_METADATA_TABLE_NAME = "facetMetadata";
+
+    public FacetQueryTestTableHelper(String instanceName, Logger log, RebuildingScannerTestHelper.TEARDOWN teardown,
+                    RebuildingScannerTestHelper.INTERRUPT interrupt)
+                    throws AccumuloSecurityException, AccumuloException, TableExistsException, TableNotFoundException {
+        super(instanceName, log, teardown, interrupt);
+    }
+
+    @Override
+    public void printTables(Authorizations auths) throws TableNotFoundException {
+        super.printTables(auths);
+        PrintUtility.printTable(client, auths, FACET_TABLE_NAME);
+        PrintUtility.printTable(client, auths, FACET_METADATA_TABLE_NAME);
+        PrintUtility.printTable(client, auths, FACET_HASH_TABLE_NAME);
+    }
+
+    @Override
+    public void dumpTables(Authorizations auths) {
+        super.dumpTables(auths);
+
+        try {
+            dumpTable(FACET_TABLE_NAME, auths);
+            dumpTable(FACET_HASH_TABLE_NAME, auths);
+            dumpTable(FACET_METADATA_TABLE_NAME, auths);
+        } catch (TableNotFoundException e) {
+            // should not happen
```

Comment:
Can you add a bit more explanation why this should not happen?

Reply:
Removed this try-catch, it wasn't needed